.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    # check issue and pull request once every day
+    - cron: '25 11 * * *'
+
+jobs:
+  invalid-stale-close:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
also, it is 30 days but in line 6 it seems to be 25?

Reply:
Line 6 demand github actions to check issues and prs at 11:25(UTC time,  namely Beijing time 19:25) **every day**. 
25 denotes minutes and has nothing to do with days.
Supposed I to change everyday check time to some midnight time according to Beijing time, e.g., 16:00(UTC time, namely 24:00, Beijing time)? @ZwwWayne

Reply:
Line 6 demand github actions to check issues and prs at 11:25(UTC time, namely Beijing time 19:25) **every day**.
25 denotes minutes and has nothing to do with days.
Supposed I to change everyday check time to some midnight time according to Beijing time, e.g., 16:00(UTC time, namely 24:00, Beijing time)? @ZwwWayne

Reply:
I misunderstood the meaning of line6, ignore this.